src/utils/handleOptions.js

```diff
@@ -18,21 +18,22 @@ const handleOptions = (
   ysuffix,
 ) => {
   options.maintainAspectRatio = maintainAspectRatio;
-  options.title.text = chartTitle;
+  options.plugins.title.text = chartTitle;
   if (startFrom !== 0) {
-    options.scales.xAxes[0].ticks.beginAtZero = false;
-    options.scales.yAxes[0].ticks.beginAtZero = false;
-    options.scales.xAxes[0].ticks.min = parseFloat(startFrom);
-    options.scales.yAxes[0].ticks.min = parseFloat(startFrom);
+    options.scales.y.beginAtZero = false;
+    options.scales.x.beginAtZero = false;
+    options.scales.x.min = parseFloat(startFrom);
+    options.scales.y.min = parseFloat(startFrom);
   } else {
-    options.scales.xAxes[0].ticks = { beginAtZero: true };
-    options.scales.yAxes[0].ticks = { beginAtZero: true };
+    options.scales.x.ticks = { beginAtZero: true };
+    options.scales.y.ticks = { beginAtZero: true };
   }
 
-  options.scales.xAxes[0].ticks.callback = value => {
-    return `${value}${xsuffix}`;
+  options.scales.x.ticks.callback = function (value) {
+    const label = this.chart.scales.x.getLabelForValue(value);
+    return `${label}${xsuffix}`;
```

Comment:
```suggestion
  options.scales.x.ticks.callback = (value) => {
    const label = this.chart.scales.x.getLabelForValue(value);
    return `${label}${xsuffix}`;
```

Reply:
`this` is undefined in arrow functions. I tried setting `const that = this` outside the function but it was undefined as well.

Reply:
what about skipping `this`? and writing directly `chart.scales.x.getLabelForValue(value);`? would that work?

Reply:
ah that's a callback function, keep it a function that's fine

Reply:
that wouldn't work. `chart` is undefined.